### PR TITLE
added files to package.json

### PR DESCRIPTION
**Pull request by CodeWriterWriter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,13 @@
   "bugs": {
     "url": "https://github.com/nodezoo/nodezoo-travis/issues"
   },
+  "files": [
+    "lib",
+    "srv",
+    "CHANGELOG.md",
+    "README.md",
+    "Dockerfile"
```
Comment (mirceaalexandru) [inline package.json]: package.json ?

**Comment by mihaidma** [comment]
@mirceaalexandru I think we should remove the Dockerfile and CHANGELOG.
What do you think?

**Comment by mirceaalexandru** [comment]
@mihaidma I agree with CHANGELOG but with Docker I think we can let that on. This is because I believe we should find a way to start the system not by cloning github repository but by installing from npm - a stable version. 

In that way we are sure the stable version is used.

I don't really like the idea to start a system based on cloned repository - which is not safe.

**Comment by mihaidma** [comment]
@CodeWriterWriter please remove the CHANGELOG and it can be merged

**Comment by CodeWriterWriter** [comment]
done

**Comment by mihaidma** [comment]
I'm sorry i just noticed that the license file is missing, please add also that

**Comment by mihaidma** [comment]
LGTM

**Pull request merged**